index.html

```diff
@@ -215,6 +214,14 @@ <h1 class="title">
                                         <a href="https://github.com/MozillaTN">GitHub</a>
                                         <a href="https://twitter.com/mozillaTN">Twitter</a>
                                     </div>
+                                    <div class="box">
+                                        <p class="subtitle">vMozilla</p>
+                                        <a href="https://t.me/joinchat/IQStrQxhC8M5S0TD3VQkOg">Telegram</a>
+                                        <a href="https://twitter.com/vMozilla">Twitter</a>
+                                        Facebook<a href="https://www.facebook.com/MozillaVIT">Page</a>
```

Comment:
This looks a bit off. One suggestion is you can add two links: "Facebook Page" and "Facebook Group"

![Screenshot 2019-05-28 at 9 16 25 AM](https://user-images.githubusercontent.com/21317964/58449497-620a5f80-8129-11e9-96a1-94a70c539dc7.png)

Reply:
![image](https://user-images.githubusercontent.com/13474314/58454445-c542c480-811e-11e9-9dc5-de80e4473c77.png)

How about superscripting? `the one in right col.`
Or Facebook[Page]()|[Group]()

Reply:
Superscripting seems fine. I think it gets the work done

Reply:
Resolved : :+1: 

- [x]  ::  SuperScript’d